### PR TITLE
Add json-schema def for parsed-events

### PR DESCRIPTION
**Pull request by dmcblue**
So I didn't fill out all the descriptions and patterns for the description of the Google Event but this should at least get us 90% of the way there.

### CODE REVIEW

File: schemas/event.schema.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+# Google Event Payload
```
Comment (dmcblue) [inline schemas/event.schema.md:1]: Notes from https://developers.google.com/calendar/api/v3/reference/events#resource

File: schemas/notes.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+```typescript
```
Comment (dmcblue) [inline schemas/notes.md:1]: Notes about the parsed-events

File: schemas/parsed-events.schema.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (dmcblue) [inline schemas/parsed-events.schema.json:1]: This is the main purpose of the ticket

**Comment by jbvandefliert** [comment]
Looks good to me, but I am not as familiar with the requirements for these changes or with the frontend application.

**Comment by kennyr87** [comment]
Moved branch to its own repo ... https://github.com/Badi-Cal/badi-event-schema

**Pull request closed without merging**